unstructured/nlp/tokenize.py

```diff
@@ -16,8 +16,9 @@
 
 CACHE_MAX_SIZE: Final[int] = 128
 
-NLTK_DATA_URL = "https://utic-public-cf.s3.amazonaws.com/nltk_data.tgz"
-NLTK_DATA_SHA256 = "126faf671cd255a062c436b3d0f2d311dfeefcd92ffa43f7c3ab677309404d61"
+NLTK_DATA_FILENAME = "nltk_data.3.8.2.tar.gz"
+NLTK_DATA_URL = "https://utic-public-cf.s3.amazonaws.com/{NLTK_DATA_FILENAME}"
+NLTK_DATA_SHA256 = "bd819bc707599c0cc84617f02375dfeaf65e54a4ed05ba8e44a8de25ed169805"
 
 
 def _raise_on_nltk_download(*args: Any, **kwargs: Any):
```

Comment:
With nltk 3.8.2, it seems monkey patching `nltk.download` is a bit intrusive ?

Reply:
Yeah we can remove that now. For earlier versions, we added this as an extra bit of protection against the `nltk.download` CVE.